service-level-indicators/spark-retrieval-success-rate.md

```diff
@@ -0,0 +1,612 @@
+
+Following the above approach, for each Retrieval Task, there are a set of Checkers who find this task among their $k$ closest and they will attempt the task in the round. We refer to the set of checkers who attempt a Retrieval Task as the “[Committee](#committee) ” for that task. Since we are hashing over a random value for each task when we create its `taskKey`, this approach mitigates against one party controlling an entire committee and dominating the honest majority consensus decision later in the protocol.
+
+### **How many tasks should each checker do per round? What value is given to $k$?**
+
+The choice of $k$ is determined by Spark protocol logic that aims to keep the overall number of Spark measurements completed by the total network per round fixed. This is important because we don’t want the number of requests that Storage Providers need to deal with go up as the number of Spark Checkers in the network increases.
+
+In each round, the [Round Retrieval Task List data object](http://api.filspark.com/rounds/current) specifies $k$ in a field called `maxTasksPerNode`. At the start of each round, the [spark-api](https://github.com/filecoin-station/spark-api) service looks at the number of measurements reported by the network in the previous round, compares it against the desired value, and adjusts both maxTaskPerNode $k$ and the length of the Round Retrieval Task List for the new round.
+
+The choices in each round for $k$ and the length of the Round Retrieval Task List also determine the size of the committee for each Retrieval Task. This is how we ensure each task is completed by at least $m$ checkers; the committee for each task is designed to include [between 40 and 100 checkers](https://www.notion.so/745b0e1020bb4000ac77acafee09e683?pvs=21) to make sure there are enough Measurements to add mitigations against malicious actors, but not too many that we load test the Storage Providers.
+
+The Spark Checkers that are members of each committee will go on to make the same [retrieval checks](#retrieval-checks) in the round. Each committee member then publishes their results to the Spark API (see [Reporting Measurements to Spark-API](#reporting-measurements-to-spark-api)), which then calculates the honest majority consensus about the result of the Retrieval Task (see [Evaluating Measurements with Spark-Evaluate](#evaluating-measurements-with-spark-evaluate)).
+
+Only the $k$ tasks that have been assigned to a checker in each round are treated as valid and included in the Spark RSR calculation, all others are ignored. This also means Spark checkers only get rewarded for their specified tasks each round.
```

Comment:
```suggestion
Only the $k$ tasks that the reproducible checker task selection algorithm picks in each round are treated as valid and included in the Spark RSR calculation, all others are ignored. This also means Spark checkers only get rewarded for their specified tasks each round.
```

Tasks aren't assigned to checkers, checkers pick tasks according to a deterministic algorithm. This might be nuanced but important.

Reply:
> This also means Spark checkers only get rewarded for their specified tasks each round.

What does this mean? If it means what I think it does (checkers only get rewarded for the tasks that they should work on), I think it can be removed as redundant

Reply:
Good fixes.  I agree the line can be removed.  I'm handling now.